packages/cycle-scripts-one-fits-all/scripts/start.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+const webpack = require('webpack')
+const WebpackDevServer = require('webpack-dev-server')
+
+const port = 8080
+process.env.NODE_ENV = 'development'
+
+const config = require('./configs/webpack.config.js')
```

Comment:
OK, I had a typo in '--flavor' and create-cycle-app downloaded the core flavor. You should add an error if there is an unknown option given